flink-end-to-end-tests/test-scripts/test_cli_api.sh

```diff
@@ -0,0 +1,155 @@
+SAVE_POINT_REGEX_EXTRACTOR=".*Savepoint stored in (.*)\\."
+
+EXIT_CODE=0
+
+function extract_job_id_from_job_submission_return() {
+    if [[ $1 =~ $JOB_ID_REGEX_EXTRACTOR ]];
+        then
+            JOB_ID="${BASH_REMATCH[1]}";
+        else
+            JOB_ID=""
+        fi
+    echo "$JOB_ID"
+}
+
+function extract_savepoint_path_from_savepoint_return() {
+    if [[ $1 =~ $SAVE_POINT_REGEX_EXTRACTOR ]];
+        then
+            SAVEPOINT_PATH="${BASH_REMATCH[1]}";
+        else
+            SAVEPOINT_PATH=""
+        fi
+    echo "$SAVEPOINT_PATH"
+}
+
+function cleanup_cli_test() {
+  stop_cluster
+  $FLINK_DIR/bin/taskmanager.sh stop-all
+
+  cleanup
+}
+
+printf "\n==============================================================================\n"
+printf "Test default job launch with non-detach mode\n"
+printf "==============================================================================\n"
+if [ $EXIT_CODE == 0 ]; then
+    eval "$FLINK_DIR/bin/flink run $FLINK_DIR/examples/batch/WordCount.jar"
+    EXIT_CODE=$?
+fi
+
+printf "\n==============================================================================\n"
+printf "Test run with complex parameter set\n"
+printf "==============================================================================\n"
+if [ $EXIT_CODE == 0 ]; then
+    eval "$FLINK_DIR/bin/flink run -m localhost:8081 -p 4 -q -d \
+      -c org.apache.flink.examples.java.wordcount.WordCount \
+      $FLINK_DIR/examples/batch/WordCount.jar \
+      --input file:///$FLINK_DIR/README.txt \
+      --output file:///${TEST_DATA_DIR}/out/result"
+    EXIT_CODE=$?
+fi
+
+printf "\n==============================================================================\n"
+printf "Test information APIs\n"
+printf "==============================================================================\n"
+if [ $EXIT_CODE == 0 ]; then
+    eval "$FLINK_DIR/bin/flink info $FLINK_DIR/examples/batch/WordCount.jar"
+    EXIT_CODE=$?
+fi
+if [ $EXIT_CODE == 0 ]; then
+    eval "$FLINK_DIR/bin/flink list"
+    EXIT_CODE=$?
+fi
+if [ $EXIT_CODE == 0 ]; then
+    eval "$FLINK_DIR/bin/flink list -s"
+    EXIT_CODE=$?
+fi
+if [ $EXIT_CODE == 0 ]; then
+    eval "$FLINK_DIR/bin/flink list -r"
+    EXIT_CODE=$?
+fi
+
+printf "\n==============================================================================\n"
+printf "Test operation on running streaming jobs\n"
+printf "==============================================================================\n"
+if [ $EXIT_CODE == 0 ]; then
+    RETURN=`$FLINK_DIR/bin/flink run -d \
+        $PERIODIC_JOB_JAR --outputPath file:///${TEST_DATA_DIR}/out/result`
+    echo "job submission returns: $RETURN"
+    JOB_ID=`extract_job_id_from_job_submission_return "$RETURN"`
+    eval "$FLINK_DIR/bin/flink cancel ${JOB_ID}"
+    EXIT_CODE=$?
+fi
+
+printf "\n==============================================================================\n"
+printf "Test savepoint for a running streaming jobs\n"
```

Comment:
I think this part can be removed. The coverage is already subsumed by `test_resume_savepoint.sh`.
What do you think?